### PR TITLE
improved docs for core.Grid

### LINKED ISSUE #69: Incomplete documentation for properties
In the documentation is not mentioned there exists properties `origin` and `delta` and existence of `grid` is mentioned only sort of accidentally.

I suggest to properly add information about public properties of the main object.

### PR DESCRIPTION
**Pull request by orbeckst**
- fix #69
- document all public attributes
- various text/syntax fixes
- fixed docs for MRC
- added forgotten import of `mrc` to core/__init__.py
- update CHANGELOG

### CODE REVIEW

**Comment by orbeckst** [comment]
Sorry, I don't know why RTD does not show up as a status check.

**Comment by IAlibay** [comment]
Just going to cycle the PR, sometimes "close and open" is enough to trigger webhooks.

**Comment by IAlibay** [comment]
There we go

File: gridData/__init__.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 # Released under the GNU Lesser General Public License, version 3 or later.
 # See the files COPYING and COPYING.LESSER for details.
 
-"""
-:mod:`gridData` -- Handling grids of data
+""":mod:`gridData` -- Handling grids of data
```
Comment (IAlibay) [inline gridData/__init__.py:6]: Flair wise I personally feel like it looks better to invert this i.e.
```suggestion
"""Handling grids of data -- :mod:`gridData`
```

File: gridData/core.py [hunk 2]
```diff
@@ -52,74 +52,140 @@ def _grid(x):
 
 
 class Grid(object):
```
Comment (IAlibay) [inline gridData/core.py:54]: grid.core shows up after formats in the list, would it make more sense to order it beforehand? (i.e. it seems like something that folks would want to know before they started dealing with different formats [if that makes sense]).

File: gridData/core.py [hunk 3]
```diff
@@ -52,74 +52,140 @@ def _grid(x):
...
+    ----------
+    grid : numpy.ndarray or str (optional)
+      Build the grid either from a histogram or density (a numpy nD
+      array) or read data from a filename.
 
-          grid,edges = numpy.histogramdd(...)
-          g = Grid(grid,edges=edges)
+    edges : list (optional)
+      list of arrays, the lower and upper bin edges along the axes
+      (such as the output by :func:`numpy.histogramdd`)
```
Comment (IAlibay) [inline gridData/core.py:77]:
```suggestion
      (same as the output by :func:`numpy.histogramdd`)
```
such -> same (might just be me but seems easier to understand)

File: gridData/core.py [hunk 4]
```diff
@@ -52,74 +52,140 @@ def _grid(x):
...
+      ``[0, 0, ..., 0]``.
 
-          g = Grid(grid,origin=origin,delta=delta)
+    delta : :class:`numpy.ndarray` (optional)
+      Either ``n x n`` array containing the cell lengths in each dimension,
+      or ``n x 1`` array for rectangular arrays.
 
-        From a saved file::
+    metadata : dict (optional)
+      a user defined dictionary of arbitrary key/value pairs
```
Comment (IAlibay) [inline gridData/core.py:88]: Capitalisation isn't consistent in these.

File: gridData/core.py [hunk 5]
```diff
@@ -52,74 +52,140 @@ def _grid(x):
...
+    delta : :class:`numpy.ndarray`
+       Length of a grid cell (spacing or voxelsize) in x, y, z
+       dimensions. This is a *1D array* with length
+       :attr:`Grid.grid.ndim`.
 
-        .. versionchanged:: 0.7.0
-           CCP4 files are now read with :class:`gridData.mrc.MRC` and not anymore
-           with the deprecated/buggy `ccp4.CCP4`
+    origin : :class:`numpy.ndarray`
+       array with the Cartesian coordinates of the coordinate system
```
Comment (IAlibay) [inline gridData/core.py:120]: Again inconsistent capitalisation.

File: gridData/core.py [hunk 6]
```diff
@@ -52,74 +52,140 @@ def _grid(x):
...
+       in order to represent high-dimensional data. When used with
+       data that represents real space densities then the **axis
+       convention in GridDataFormats** is that axis 0 corresponds to
+       the Cartesian :math:`x` component, axis 1 corresponds to the
+       :math:`y` component, and axis 2 to the :math:`z` component.
 
-        .. versionchanged:: 0.5.0
-           New *file_format* keyword argument.
+    delta : :class:`numpy.ndarray`
+       Length of a grid cell (spacing or voxelsize) in x, y, z
```
Comment (IAlibay) [inline gridData/core.py:115]: use ":math:`x`" etc.. here too?

File: gridData/core.py [hunk 7]
```diff
@@ -452,12 +521,9 @@ def _load(
                     grid, edges, origin, delta))
 
     def load(self, filename, file_format=None):
-        """Load saved (pickled or dx) grid and edges from <filename>.pickle
+        """Load saved grid and edges from `filename`
```
Comment (IAlibay) [inline gridData/core.py:524]: Just checking that you want italics here and not double ticks around the argument name.

Comment (IAlibay) [reply gridData/core.py:524]: Same for all the other below.

**Review by IAlibay — APPROVED** [review]
lgtm, thanks! (couple of comments but nothing blocking)

File: gridData/__init__.py (hunk 1 above)
Comment (orbeckst) [reply gridData/__init__.py:6]: changed

File: gridData/core.py (hunk 2 above)
Comment (orbeckst) [reply gridData/core.py:54]: changed order

File: gridData/core.py (hunk 3 above)
Comment (orbeckst) [reply gridData/core.py:77]: yes, oops, changed

File: gridData/core.py (hunk 4 above)
Comment (orbeckst) [reply gridData/core.py:88]: all capitalized

File: gridData/core.py (hunk 5 above)
Comment (orbeckst) [reply gridData/core.py:120]: capitalized

File: gridData/core.py (hunk 6 above)
Comment (orbeckst) [reply gridData/core.py:115]: changed

File: gridData/core.py (hunk 7 above)
Comment (orbeckst) [reply gridData/core.py:524]: Single tics ("no role") seems to be the standard when referring to arguments.

**Comment by orbeckst** [comment]
thanks for the quick turnaround @IAlibay

**Pull request merged**